requirements.txt

```diff
@@ -1,2 +1 @@
-aiohttp
-pygit2
+.
```

Comment:
maybe we can maintain the file as it is, or rename it to requirement-devel.txt. I am not sure what this point means

Reply:
The dot instructs pip to look into `setup.py` for dependencies, kinda like a symlink. I can rollback if you find it more useful to have it there, but it's duplicated in this case.

`pip install -r requitements.txt` still works and install these, but you will notice a message that pip is actually looking somewhere else for packages.